### DISCUSSION TITLE
I need to read a cell with HTML

### DISCUSSION THREAD

**Discussion by brothman-unbridled** [PHPOffice/PhpSpreadsheet#2812] [Q&A] [answered]
Obviously this plugin works beautifully for everything else, but I have a cell with HTML and text in it but when PhpSpreadsheet returns input for that cell it returns sanitized text without the HTML code that I need in it.  Can I tell PhpSpreadsheet to read that entire cell?

**Comment by oleibman** [comment]
I don't see any problem when I attempt that. Perhaps there is something about your spreadsheet and/or code which triggers a problem which my test doesn't encounter. Can you upload those?

**Reply by brothman-unbridled** [reply, marked as answer]
hmm ok so you are saying something else in the code is sanitizing the input and it is not an inherent part of PhpSpreadsheet... that actually puts me on the right track, thanks
